### DISCUSSION TITLE
Enunciado: mismo o igual?

### DISCUSSION THREAD

**Discussion by clangoi** [IIC2133-PUC/2024-1#178] [Tarea 3] [answered]
Hola, solo para confirmar, aqui deberia decir distinto o igual cierto?
![image](https://github.com/IIC2133-PUC/2024-1/assets/53914794/efd00a70-ffc8-4b9b-9c64-9e13e128ede3)

**Comment by benjavicente** [comment, marked as answer]
Sip, distinto o igual.
